src/uct/ib/ud/base/ud_ep.c

```diff
@@ -244,7 +244,9 @@ static unsigned uct_ud_ep_deferred_timeout_handler(void *arg)
     status = iface->super.ops->set_ep_failed(&iface->super, &ep->super.super,
                                              UCS_ERR_ENDPOINT_TIMEOUT);
     if (status != UCS_OK) {
-        ucs_fatal("UD endpoint %p: unhandled timeout error", ep);
+        ucs_fatal("UD endpoint %p to "UCT_UD_EP_PEER_NAME_FMT": "
```

Comment:
it seems defenitions of `UCT_UD_EP_PEER_NAME_FMT` and `UCT_UD_EP_PEER_NAME_ARG` weren't added in this PR